tests/algorithms/test_bc.py

```diff
@@ -186,7 +186,7 @@ def test_that_bc_improves_rewards(
     cartpole_venv: vec_env.VecEnv,
 ):
     # GIVEN
-    # TODO: the upstream annotation for this function is overly-conservative
+    # TODO(GH#600): the upstream annotation for this function is overly-conservative
```

Comment:
May I suggest
```suggestion
    # TODO(https://github.com/HumanCompatibleAI/imitation/issues/600): the upstream annotation for this function is overly-conservative
```
Easier to click?

Reply:
Makes line too long but it's a multi-line comment so sure I'll just reflow it.